apps/ui/lib/components/Event/Message/DisplayReactions.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import emoji from 'node-emoji';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import { Button, Txt } from 'rendition';
+import { useSetup } from '../../SetupProvider';
+import { selectors } from '../../../store';
+import { Contract } from 'autumndb';
+
+const Emoji = ({ symbol, count, ...rest }) => {
+	const em = React.useMemo(() => {
+		return emoji.emojify(symbol);
+	}, [symbol]);
+
+	return (
+		<Txt {...rest}>
+			{em}
+			<span>{count}</span>
+		</Txt>
+	);
+};
+
+const Wrapper = styled.section`
+	display: flex;
+	flex-direction: row;
+	align-content: space-between;
+	width: 200px;
+`;
+
+export const DisplayReactions = ({ message }: { message: Contract }) => {
+	const currentUser = useSelector(selectors.getCurrentUser());
+
+	const reactions = message.links?.['has attached element']?.filter(
+		(contract) => {
+			return contract.type === 'reaction@1.0.0';
+		},
+	);
+
+	const [groupReactions, setGroupReactions] = useState(
+		{} as { [key: string]: Contract[] },
+	);
+
+	const { sdk } = useSetup()!;
+	// TODO: Remove this workaround when this is resolved
+	// https://github.com/product-os/autumndb/issues/1333
+	React.useEffect(() => {
+		if (!reactions) {
+			return;
+		}
+		const ac = new AbortController();
+		sdk
+			.query({
+				type: 'object',
+				$$links: {
+					'was created by': {
+						type: 'object',
+					},
+				},
+				properties: {
+					id: {
+						enum: reactions.map((reactionContract) => reactionContract.id),
```

Comment:
It's possible for `reactions` to be an empty array. You should check for this and exit early if there are no reactions

Reply:
In line 48, we are checking if reactions is empty and return on that condition